README.md

```diff
@@ -99,9 +99,25 @@ Installing
 
 -	`yarn install` to install the local dependencies
 
-### Configuring
+### Buildtime Configuring
```

Comment:
I would restoring the old `### Configuring` header, and moving the `Buildtime Configuring` as a subheader, listing it **after** the `Runtime Configuring`.   The reason I'd recommend listing Runtime Configuring **first** is because that is the configuration most everyone will use.... the Buildtime configs are only for advanced configuration.

Reply:
Will do.

Reply:
Done.